src/detect.c

```diff
@@ -199,6 +199,8 @@
 
 #include "runmodes.h"
 
+#include <glob.h>
```

Comment:
Ok, will try to handle it. If header is not found, suricata should compile anyway, but without wildcard support?